server/apps/main/templates/main/moderate_experience.html

```diff
@@ -155,7 +155,7 @@ <h3>What could have made the experience better?</h3>
         <p><i>{{experience_difference}}</i></p>
       </div>
 
-      <template id="template-control" style="display: none;">
+      <template id="template-control" class="template-hidden">
```

Comment:
I only today learned about the `template` tag existing, super cool!

Reply:
AutSPACEs taught me about it too 😃